ROMFS/px4fmu_common/init.d/50002_traxxas_stampede_2wd

```diff
@@ -0,0 +1,79 @@
+#!nsh
+#
+# @name Traxxas stampede vxl 2wd
+#
+# @url https://traxxas.com/products/models/electric/stampede-vxl-tsm
+#
+# @type Rover using Flying Wing configuration
+#
+# 
+# @output MAIN2 steering
+# @output MAIN4 throttle
+#
+# @output AUX1 feed-through of RC AUX1 channel
+# @output AUX2 feed-through of RC AUX2 channel
+# @output AUX3 feed-through of RC AUX3 channel
+#
+# @maintainer Marco Zorzi
+#
+
+sh /etc/init.d/rc.gnd_defaults
+
+if [ $AUTOCNF == yes ]
+then
+	param set BAT_N_CELLS 7
+	
+	param set FW_MAN_R_MAX 90
+
+	param set FW_YR_FF 0.0
+	param set FW_YR_I 0.0
+	param set FW_YR_IMAX 0.0
+	param set FW_YR_P 0.0
+
+	param set FW_THR_CRUISE 2.0
+	param set FW_THR_IDLE 0.0
+	param set FW_THR_LND_MAX 20.0
+	param set FW_THR_MAX 5.0
+
+	param set FW_AIRSPD_MIN 0
+	param set FW_AIRSPD_TRIM 1
+	param set FW_AIRSPD_MAX 3
+	
+	param set NAV_ACC_RAD 0.5
+	param set NAV_LOITER_RAD 2
+
+	param set MIS_LTRMIN_ALT 0.01
+	param set MIS_TAKEOFF_ALT 0.01
+	
+	param set EKF2_GBIAS_INIT 0.01
+	param set EKF2_ANGERR_INIT 0.01
+	param set EKF2_MAG_TYPE 1
+
+	param set GND_WR_P 2
+	param set GND_WR_I 0.9674
+	param set GND_WR_IMAX 0.1
+	param set GND_WR_D 1.2
+	param set GND_SP_CTRL_MODE 1
+	param set GND_L1_DIST 10
+	param set GND_THR_IDLE 0
+	param set GND_THR_CRUISE 0
+	param set GND_THR_MAX 0.5
+	param set GND_THR_MIN 0
+	param set GND_SPEED_P 0.25
+	param set GND_SPEED_I 0.001
+	param set GND_SPEED_D 3
+	param set GND_SPEED_IMAX 0.125
+	param set GND_AIRSP_THR_SC 1
+
+fi
+
+# Configure this as plane
+set MAV_TYPE 1
```

Comment:
This should be `MAV_TYPE_GROUND_ROVER`, which means it should be set to `10`, not  `1`.